samba/rootfs/etc/cont-init.d/samba.sh

```diff
@@ -24,6 +24,11 @@ bashio::log.info "Hostname: ${HOSTNAME}"
 for interface in $(bashio::network.interfaces); do
     interfaces+=("${interface}")
 done
+# Add default interface if it is not part of the supported interfaces list
```

Comment:
I've asked people who reported this issue to provide the output. It seems they somehow created an installation where `bashio::network.name` is not part of `bashio::network.interfaces`. My knowledge about supervisor is so minimal, I don't know how/whether it is possible.

Reply:
Ludeeus is right. I don't understand how it is possible but if its possible its a bug. `bashio::network.interfaces` calls `/api/network/info` and pulls out the interfaces, that list of interfaces comes from [here](https://github.com/home-assistant/supervisor/blob/main/supervisor/host/network.py#L64). `bashio::network.name` loops through that same list of interfaces and finds the one specified as `primary`.

So I'm not really seeing how its possible for this to fix the issue. But if it does then we have a bug to fix in either supervisor or bashio which means it would be a temporary fix at best. Which means its not a good idea for us to accept this.

Sorry I didn't think to check this when you suggested the idea @lmagyar . Didn't mean to waste your time.